### ISSUE TITLE
Update on Dockerhub?

### ISSUE THREAD

**Issue by poVoq** [QuantumObject/docker-ushahidi-platform-client#4]
Any chance a new release based on 18.04 and the latest Ushahidi could be uploaded to Dockerhub?

Thanks!

**Comment by poVoq** [comment]
Ok I see they were updated on dockerhub, but not yet here on github.
Did you switch to the "production" branch or did you manage to get the new "develop" branch working?

**Comment by angelrr7702** [comment]
yes. I still have to fix the platform ... client need to wait after that ...

**Comment by angelrr7702** [comment]
it fail to install under ubuntu 18.04 because problem php7.2 that don't support "The requested PHP extension ext-mcrypt * is missing from your system" and waiting for ushahidi take care for modifying their software. php-mcrypt is not maintained and no supported and with some security issue . 

at the same time the original version ubuntu 16.04 fail to rebuild my container , checking what fail .

**Comment by angelrr7702** [comment]
quantumobject/docker-ushahidi-platform-client  was update to Ubuntu 18.04 and last client . like always need to wait few min to finish installing process for the client to work . 

I still have problem with quantumobject/docker-ushahidi-platform (platform api) .  please used the other repository to create issue relate to platform api .

**Comment by poVoq** [comment]
Great thanks.

Any chance you could look into Dockerizing this as well?
https://github.com/gvSIGAssociation/gvsig-online
It is IMHO the better solution than Ushahidi for many applications and the mobile client is also open source.
Thanks!

**Comment by angelrr7702** [comment]
update , need to check if problem still present
